docs/guides/high-availability/operate-site-offline.adoc

```diff
@@ -0,0 +1,78 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Take site offline"
+summary="This guide describes how to take a site offline so that it no longer processes client requests" >
+
+== When to use this procedure
+
+During the deployment lifecycle it might be required that one of the sites is temporarily taken offline
+for maintenance or to allow for software upgrades.
+
+== Procedure
+
+Follow these steps to remove a site from the AWS Global Accelerator so that no traffic can be routed to it.
+
+=== Global Accelerator
```

Comment:
As above: try to separate the concept and the blueprint specific implementation. This also applies for the other operational procedures.